src/main/CommandHandler.cpp

```diff
@@ -969,7 +970,12 @@ CommandHandler::maintenance(std::string const& params, std::string& retStr)
     http::server::server::parseParams(params, map);
     if (map["queue"] == "true")
     {
-        mApp.maintenance(50000);
+        auto count = 50000;
```

Comment:
you should use `uint32` instead of auto, otherwise you're allowing people to pass in negative numbers

Reply:
talking about that, it looks like `parseNumParam` has a bug: it doesn't check if the stream failed when parsing the input `str.fail()`